src/components/VolumeAlertForm/VolumeAlertForm.tsx

```
@@ -4,6 +4,12 @@ import { useDebouncedCallback } from 'use-debounce';
 import { PlugZap, Bot } from "lucide-react";
 import { Spinner } from "@/components/ui/spinner";
 
+import React from "react";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)